arch/arm/arm_features.c

```diff
@@ -11,6 +11,7 @@
 #    define ID_AA64ISAR0_CRC32_VAL ID_AA64ISAR0_CRC32
 #  endif
 #elif defined(__APPLE__)
+#  define _DARWIN_C_SOURCE /* enable types aliases (eg u_int) */
```

Comment:
This might break if zlib-ng is included by another project... Like other similar preprocessor defines, it should always be guarded.

Reply:
sorry I missed your comment.